src/config.ts

```diff
@@ -1 +1,11 @@
 export const INFURA_API_KEY = "92c9a51428b946c1b8c1ac5a237616e4";
+
+// DID Universal Resolver parameters
+export const BASE_RESOLVER_URL = "https://uniresolver.io";
+export const RESOLVER_VERSION = "1.0";
+export const RESOLVER_PATH = "identifiers";
+
+// w3c-dids
+export const ETHR_DID_METHOD = "ethr";
+export const SUPPORTED_DID_AUTH = ["Secp256k1SignatureAuthentication2018", "Secp256k1SignatureAuthentication2019"];
```

Comment:
no 2020, no 2017, etc. ?

sorry I'm noob on this, what's the diff between 2018/2019 ? just the name or there is more ?

Reply:
no 2020 as yet.  I believe there were slight differences in 2019 as drafted here: (https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/).  I am not certain how versioning occurs / naming though.